src/main/java/org/datacleaner/kettle/configuration/DataCleanerConfigurationDialog.java

```diff
@@ -189,11 +186,7 @@ public void widgetSelected(SelectionEvent event) {
                 labelVersion.setText(VERSION.trim() + " " + editionDetails.getVersion());
             }
         } catch (Throwable e) {
-            String errorMessage = "There was an unexpected error launching DataCleaner";
-            if (e instanceof IOException) {
-                errorMessage = "The DataCleaner installation path could not be found.Please set the path in the menu Tools:DataCleaner configuration";
-            }
-            new ErrorDialog(Spoon.getInstance().getShell(), "Error launching DataCleaner", errorMessage, e);
+           //Do nothing 
```

Comment:
Couldn't this also use the JOptionPane? No error message seems dangerous. Or is it handled in the ModelerHelper (should that be _Modal_ Helper, btw?)? In that case I would remove that try/catch block completely, it's not very nice to swallow exceptions.